src/web/docusaurus/docs/architecture.md

```diff
@@ -52,7 +36,7 @@ The legacy back-end has been split into a series of microservices. Each microser
 - Posts Service (`src/api/posts`) - API for accessing Post and Feed data in Redis (probably not well named at this point)
 - Search Service (`src/api/search`) - API for doing searches against Elasticsearch
 - Status Service (`src/api/status`) - API for accessing Telescope status information, as well as providing the Dashboards
-- Parser Service (`src/api/parser`) - feed and post parsing. Currently disabled, see <https://github.com/Seneca-CDOT/telescope/issues/2111>
+- Parser Service (`src/api/parser`) - feed and post parsing.
```

Comment:
What if we clarify that the Parser service was disabled, and we add a new section for release 3.0 where the parser service was enabled, and the legacy backend was completely removed?